context/brand-context/springernature/README.md

```diff
@@ -0,0 +1,22 @@
+## Icons
+
+Recommendation of using icons is the same as in the 
+[Icons section of the default brand context](../default/README.md#icons)
+
+In case you are generating SVG sprites in combination with
+[SVGO](https://github.com/svg/svgo), be wary of the `cleanupIDs` configuration
+which is enabled by default. The configuration minifies IDs which may lead to ID
+duplication in the HTML document. This duplication fails [W3C
```

Comment:
Suggested rewording:
>This duplication fails W3C validation and violates WCAG 2.1 [SC 4.1.1 Parsing](https://www.w3.org/TR/WCAG21/#parsing) (see [Techniques & Failures F77](https://www.w3.org/WAI/WCAG21/Techniques/failures/F77.html) to understand why).

* F77 is an informative document, not a normative one - it's the WCAG criterion itself that's being violated, and F77 just explains why. 
* I updated the links to WCAG 2.1 instead of WCAG 2.0 😄 

Nicely done!

Reply:
Thank you so much @hollsk 🙏 .
I've committed the copy change. Shall I merge this in now?